network/basics/uri.py

```diff
@@ -37,7 +36,7 @@
 description:
   - Interacts with HTTP and HTTPS web services and supports Digest, Basic and WSSE
     HTTP authentication mechanisms.
-version_added: "1.1"
+version_added: "1.9"
```

Comment:
This `version_added` should not be changed, it was the version that the `uri` module was first added in